codelists/api.py

```diff
@@ -48,15 +52,22 @@ def codelists_get(request, owner):
     ]
     """
 
+    filter_kwargs = {}
+
+    if "coding_system_id" in request.GET:
+        filter_kwargs["coding_system_id"] = request.GET["coding_system_id"]
+
     records = []
 
     for cl in sorted(
-        owner.codelists.prefetch_related("handles", "versions"), key=lambda cl: cl.slug
+        owner.codelists.filter(**filter_kwargs).prefetch_related("handles", "versions"),
```

Comment:
If someone passed in "coding_system_id": "" would that cause unexpected behaviour here? Might be worth checking that the value is truthy in the if-statement above?